src/app/tutorials/tutorials.component.spec.ts

```diff
@@ -1,14 +1,16 @@
 import { async, ComponentFixture, TestBed } from '@angular/core/testing';
 
 import { TutorialsComponent } from './tutorials.component';
+import { RouterTestingModule } from '@angular/router/testing';
 
 describe('TutorialsComponent', () => {
   let component: TutorialsComponent;
   let fixture: ComponentFixture<TutorialsComponent>;
 
   beforeEach(async(() => {
     TestBed.configureTestingModule({
-      declarations: [ TutorialsComponent ]
+      declarations: [ TutorialsComponent ],
+      imports: [RouterTestingModule],
```

Comment:
To test components that use `router-outlet` you need to import `RouterTestingModule`, a module specifically for tests.